uavcan/diagnostic/65520.Text.1.0.uavcan

```diff
@@ -0,0 +1,15 @@
+#
+# Generic human-readable text message for logging and displaying purposes.
+# Generally it should be published at the lowest priority level.
+#
+
+# Standard severity, 2 bit wide.
+Severity.1 severity
```

Comment:
I wasn't quite happy about LogLevel because of the implication that the text can be used only for logging purposes, whereas there is another valid use case related to real-time diagnostic displays. So I searched for a better name that would suit both use cases simultaneously without any bias towards either, and I ended up with Severity. The word `TRIVIAL` has the same origins (the other three were chosen to appease people with an aerospace background after we discussed that with Scott). I am open to better names, but I would prefer staying away from anything related to logging, if possible.

Reply:
Are there any advantages to using the same data types for both these use cases?

Reply:
I don't see any benefit in their separation. Logging and indication seem to be highly related.

Reply:
With the current suggestion for this message, logging becomes more confusing. It's not trivial to know that `uavcan.diagnostics.Text` is a log message. And it becomes harder when you have to map some weird sounding (except for people with aerospace background) "Severities" to what you think of as log levels.

Perhaps if you elaborate on how this RT diagnostic display and how it relates to these severities it would help me see the whole picture? If you're diagnosing RT properties I think the `Trace` log level would be the intended way to do this. But then I guess the log message should have an optional timestamp as well?

Reply:
I have removed this namespace from this PR to be discussed separately.